test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -321,7 +321,7 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 				containerCgroup, err = nodes.ExecCommandOnNode(cmd, workerRTNode)
 				Expect(err).ToNot(HaveOccurred())
 				return containerCgroup
-			}, (cluster.ComputeTestTimeout(30*time.Second, RunningOnSingleNode)), 5*time.Second).ShouldNot(BeEmpty(),
+			}, cluster.ComputeTestTimeout(30*time.Second, RunningOnSingleNode), 5*time.Second).ShouldNot(BeEmpty(),
```

Comment:
since we are changing this, let's pivot to use `WithTimeout` and `WithPolling` (may be relevant also elsewhere)

Reply:
Done. Thanks